.github/workflows/misc.yml

```diff
@@ -1,4 +1,4 @@
-name: Miscellaneous - Linux
+name: Miscellaneous review SHA
```

Comment:
Was this intentional?

Reply:
Doh, no I'm not sure how that happened. Thanks for catching!